edgelet/edgelet-hsm/tests/crypto_get_random.rs

```diff
@@ -2,6 +2,7 @@
 
 #![deny(unused_extern_crates, warnings)]
 #![deny(clippy::all, clippy::pedantic)]
+#![allow(clippy::similar_names)]
```

Comment:
from https://rust-lang.github.io/rust-clippy/master/ and filter by "similar_names", it shows that similar_names rule is "Checks for names that are very similar and thus confusing".  I think we should fix it instead of disable it.  Does this issue happen in master.  I think Mike asked in teams, but no one responds.  Please reply in teams and confirm no one is currently working on it.

Reply:
Arnav suggested to suppress.  There are already a few of allowing the ` similar_names` in the master.

Reply:
@arsing since we disable at file level, it will not detect when any new changes fail with this rule.  If so, should we just disable at global level instead of at file level?

Reply:
This is the global level. Files under `tests/` are integration tests. Each file is compiled as a crate.